### PR TITLE
Issue/3

### LINKED ISSUE #3: Cannot run under Python 3.5.2 - "TypeError: 'generator' object is not subscriptable"
When running the server under Python 2.7.12, everything seems to work. When running it under 3.5.2 and trying to push documentation using the exact same commands, the server returns HTTP 500 (internal server error), and it prints this on console:

```
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\davidfallah\AppData\Local\Programs\Python\Python35-32\lib\site-packages\flask\app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\davidfallah\Documents\hostthedocs\hostthedocs\__init__.py", line 22, in hmfd
    request.files.values()[0].stream,
TypeError: 'generator' object is not subscriptable
```
 
I changed `request.files.values()[0].stream` to `list(request.files.values())[0].stream` and this seems to make it work under both Python 2.7.12 and 3.5.2. I have not tested for other versions.

### PR DESCRIPTION
**Pull request by TAGC**
First substantial pull request I've ever done but let's give it a shot.

Fixes #3

### CODE REVIEW

File: tests/test_util.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            # THEN the file-stream should correspond to the provided file.
+            self.assertEqual(content, filestream.read())
+
+    @raises(ValueError)
+    def test_exception_thrown_if_request_contains_no_file(self):
+        # GIVEN a request without any files.
+        builder = EnvironBuilder(method='POST')
+        request = Request(builder.get_environ())
+
+        # EXCEPT an exception is thrown when we process the request.
```
Comment (TAGC) [inline tests/test_util.py]: Typo.

File: hostthedocs/__init__.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 from flask import abort, Flask, jsonify, redirect, render_template, request
 
+from hostthedocs import util
```
Comment (rgalanakis) [inline hostthedocs/__init__.py]: Let's do `from . import util` to keep it consistent.

**Comment by rgalanakis** [comment]
Really great job on this- just a minor style thing. 😗👌

**Comment by rgalanakis** [comment]
Also, would be good to squash the typo commit into the commit that introduced the typo, no need for that as a distinct commit.

File: hostthedocs/__init__.py (hunk 2 above)
Comment (TAGC) [inline hostthedocs/__init__.py]: Well spotted.

**Pull request merged**